.travis.yml

```diff
@@ -1,8 +1,9 @@
 language: node_js
 
 node_js:
-  - "6"
   - "8"
+  - "10"
+  - "12"
   - "node"
 script: "npm run-script test-travis"
 after_success: 'npm run coveralls'
```

Comment:
i'd recommend running the `after_success` only for the latest version.
see an example here https://github.com/PayU/prometheus-api-metrics/blob/master/.travis.yml